test/server/cards/01_SOR/units/RedThreeUnstoppable.spec.js

```diff
@@ -0,0 +1,37 @@
+describe('Red Three', function () {
+    integration(function () {
+        describe('Red Three\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['death-trooper', 'bail-organa#rebel-councilor', 'battlefield-marine'],
+                        spaceArena: ['red-three#unstoppable', 'green-squadron-awing']
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors', 'cargo-juggernaut']
+                    }
+                });
+            });
+
+            it('should give Raid 1 to heroism unit', function () {
+                this.player1.clickCard(this.battlefieldMarine);
+                expect(this.player1).toBeAbleToSelectExactly([this.p2Base, this.ruggedSurvivors, this.cargoJuggernaut]);
+                this.player1.clickCard(this.p2Base);
+                expect(this.p2Base.damage).toBe(4);
+            });
+
+            it('should not give Raid 1 to non-heroism unit', function () {
```

Comment:
Let's consolidate all three of these into one test, you can look at the tests for Iden Versio as an example. In cases where it is possible to have fewer test cases we do that to reduce the use of github actions time (although it is not good test practice, sadly).